docs/using_pyodide_from_webworker.md

```diff
@@ -10,83 +10,171 @@ Setup your project to serve `webworker.js`. You should also serve
 `pyodide.js`, and all its associated `.asm.js`, `.data`, `.json`, and `.wasm`
 files as well, though this is not strictly required if `pyodide.js` is pointing
 to a site serving current versions of these files.
+The simplest way to serve the required files is to use a CDN, 
+such as [pyodide-cdn2.iodide.io][cdn pyodide]. This is the solution 
```

Comment:
```suggestion
such as `https://cdn.jsdelivr.net/pyodide`. This is the solution 
```

There is probably need create a link there, as the index page will return 404 not found anyway.